scripts/all-tests.sh

```diff
@@ -191,6 +191,7 @@ END
                 # May be unnecessary if we changed the "Skip if there are no test files"
                 # option to True in the jenkins job definitions.
                 mkdir -p reports
+                mkdir -p reports/bok_choy
```

Comment:
Shouldn't just changing `mkdir -p reports` to `mkdir -p reports/bok_choy` work instead of doing both?